face_track/face_track.py

```diff
@@ -24,7 +24,7 @@
 import math
 import logging
 
-from std_msgs.msg import Int32
+from std_msgs.msg import Int32, String
 from pi_face_tracker.msg import FaceEvent, Faces
```

Comment:
given that the pi_vision face tracker will someday be replaced by a cppmt face tracker, I think its a bad idea to add sound the the pi_vision subsystem.

You really want sound handled by a different subsystem, so that the port to the cmt face tracker is easier.